web/pgadmin/static/js/helpers/Notifier.jsx

```diff
@@ -175,29 +189,38 @@ class Notifier {
     this.modal.alert(title, text, onOkClick, okLabel);
   }
 
-  confirm(title, text, onOkClick, onCancelClick, okLabel=gettext('Yes'), cancelLabel=gettext('No'), extras=null) {
+  confirm(title, text, onOkClick, onCancelClick, okLabel=gettext('Yes'), cancelLabel=gettext('No')) {
     /* Use this if you want to use pgAdmin global notifier.
     Or else, if you want to use modal inside iframe only then use ModalProvider eg- query tool */
-    this.modal.confirm(title, text, onOkClick, onCancelClick, okLabel, cancelLabel, extras);
+    this.modal.confirm(title, text, onOkClick, onCancelClick, okLabel, cancelLabel);
   }
 
-  confirmDelete(title, text, onOkClick, onCancelClick,okLabel = gettext('Yes'), cancelLabel = gettext('No')){
-
-    const  extraButtons = (closeModal) => {
-      return [
-        {
-          type: 'default',
-          icon: <CheckRoundedIcon />,
-          label: okLabel,
-          onClick: ()=>{
-            onOkClick();
-            closeModal();
-          },
-          color: 'error',
-        },
-      ];
-    };
-    this.modal.confirm(title, text, onOkClick, onCancelClick, okLabel, cancelLabel, extraButtons);
+  confirmDelete(title, text, onOkClick, onCancelClick, okLabel = gettext('Delete'), cancelLabel = gettext('Cancel')){
+    let isDeleteButton = gettext('Delete') === okLabel;
```

Comment:
In a dialog type `confirmDelete` focus should by default be on Cancel. Checking based on label better be avoided.